docs/publications.rst

```diff
@@ -25,7 +25,7 @@ If you use *fiction* in your work, we would appreciate if you cited
     }
 
 Furthermore, if you use any of the design automation algorithms :ref:`exact <exact>`, :ref:`ortho <ortho>`,
-:ref:`onepass <onepass>`, :ref:`equiv <equiv>`, :ref:`color routing <color_routing>`, or :ref:`QuickSim <quicksim>`
+:ref:`onepass <onepass>`, :ref:`equiv <equiv>`, :ref:`color routing <color_routing>`, :ref:`QuickSim <quicksim>`, or :ref:`Critical Temperature <critical_temperature>`
```

Comment:
This reference leads nowhere. Could you please add a respective anchor in the RST file?

Reply:
Thank you very much! I think I had a typo in the anchor.